cfgov/jinja2/v1/_includes/macros/category-icon.html

```diff
@@ -27,7 +27,13 @@
         'op-ed': 'cf-icon-contract',
         'testimony': 'cf-icon-double-quote',
         'blog': 'cf-icon-speech-bubble',
-        'newsroom': 'cf-icon-newspaper'
+        'newsroom': 'cf-icon-newspaper',
+        'featured event': 'cf-icon-date',
+        'featured blog': 'cf-icon-speech-bubble',
+        'featured video': 'cf-icon-play-round',
+        'featured tool': 'cf-icon-settings',
+        'featured news': 'cf-icon-newspaper',
+        'featured': 'cf-icon-favorite'
       }
```

Comment:
Can you add a todo to the top of this logic to eventually move it to the backend? The slug really should be passed in with the value instead.